### ISSUE TITLE
Icon for Jamtaba

### ISSUE THREAD

**Issue by jtrozos** [elieserdejesus/JamTaba#1148]
Hi, Just wanted to suggest submissions for a new Jamtaba icon. Just an unpolished idea:

![jamtaba2](https://user-images.githubusercontent.com/30012149/48669794-f907b680-eb0b-11e8-9dab-887b16fb2711.jpg)

I would like you to consider removing some themes and update the screenshots in the website using the flat theme. Thanks!

**Comment by fab672000** [comment]
The icon looks okay but personally, I'm not a big fan of icons with a large app text name on it ; and I must confess I like the original picture for JamTaba (the text is also there in the original but a bit smaller ).

**Comment by jtrozos** [comment]
I totally agree with you about adding texts to the icon but as you can see in the current icon the word "play" is unintelligible being that small and "Jamtaba" is being croppped in circle shaped icons pages like facebook so it would be great if the best realtime online jamming app has at least an updated design.

![comparison](https://user-images.githubusercontent.com/30012149/48700880-6b03fc80-ebed-11e8-9172-27cdc3e775f4.jpg)

I'm not a designer so I'm just throwing ideas. It would be great if you, community, could bring a new icon.

![jam4](https://user-images.githubusercontent.com/30012149/48701464-dd291100-ebee-11e8-80c3-e8888ce2c0b4.jpg)

![jam6](https://user-images.githubusercontent.com/30012149/48701524-021d8400-ebef-11e8-9768-c6278d0ab03c.jpg)

**Comment by fab672000** [comment]
I definitely like no icon text better but then i miss the 3D shading on the original, while appreciating the jacks idea which I think is good! the icon of jamtaba you show is not the one i am familiar with on OSX we have this nice one:
![jtb_osx_icon](https://user-images.githubusercontent.com/6124901/48714533-81538d80-ebd8-11e8-8b74-f824f1339a60.png)
